### PR TITLE
Add missing docs

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #299 and #303

### CODE REVIEW

File: docs/src/user/guide/forgot_password.md
Comment (Jeremy-Prior) [inline docs/src/user/guide/forgot_password.md:1]: The content is good but please look at some of the other guide docs to see the preferred formatting. This should be different to the manual.

File: docs/src/user/guide/logout.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Logging out of the account is a simple process that ensures the security of your session. Here's a quick guide to help the user through the logout procedure:
+
+Click on the user profile icon or name to reveal the user-related options.
+
+![logout option](../manual/img/logout-1.png)
+
+In the dropdown menu, click on the `Logout` option. After clicking `Logout`, the system will process the request and redirect the user to the logout page.
+
+**Logout Page**
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/logout.md:11]:
```suggestion
## Logout Page
```

File: docs/src/user/guide/logout.md
Comment (Jeremy-Prior) [inline docs/src/user/guide/logout.md:1]: The content is good but please look at some of the other guide docs to see the preferred formatting. This should be different to the manual.

File: docs/src/user/manual/index.md [hunk 2]
```diff
@@ -16,3 +16,7 @@ users may prefer to work through our [user guide](../guide/index.md).
 * **[About:](./about.md)** This is the `About` page where users can learn more about the site.
 * **[Help:](./help.md)** The help page is the documentation site.
 * **[User Roles:](./user-roles.md)** Access to data and functionalities on the SAEOSS-Portal platform is governed by user roles and permissions.
+* **[SignUp:](./signup.md)** This is the `SignUp` page where the user can create an account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/index.md:19]:
```suggestion
* **[Sign Up:](./signup.md)** This is the `Sign Up` page where the user can create an account.
```

File: docs/src/user/manual/login.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Login Page User Manual
+
+## Overview
+
+The Login page is the gateway to accessing the features and content on the platform. This guide will walk the users through the steps to log in, create an account, and recover your password.
+
+![login options](./img/login-1.png)
+
+**Login Buttons:**
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login.md:9]:
```suggestion
### Login Buttons
```

File: docs/src/user/manual/login.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Users can access the login functionality from two locations on the home page.
+
+**Access Login Page:**
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login.md:13]:
```suggestion
```

File: docs/src/user/manual/login.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+1. **Login Button:** Click on the login button available in the middle of the home page. Alternatively, locate and click on the login button in the top right corner of the page.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/login.md:15]: We should probably separate the second login button. e.g.
```suggestion
1. **Login Button:** Click on the login button available in the middle of the home page.
2. **Other Login Button:** Alternatively, locate and click on the login button in the top right corner of the page.
```

File: docs/src/user/manual/logout.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+1. **Back to Home Page Button:** Click on this button to return to the home page, providing a quick way to access essential information and features.
+
+2. **Metadata:** Navigate to the `Metadata` section by clicking on this button. Explore and discover valuable metadata records.
+
+3. **Map:** Access the `Map` section by clicking on this button. Visualise spatial Earth Observation datasets and interact with the map interface.
+
+## Conclusion
+
+The Logout option ensures a secure end to the user's session, redirecting the user to a dedicated logout page. From there, the user can conveniently navigate to different sections, including the home page, login page, and signup page, using the provided buttons. Feel free to explore the various features available after logging out and return to the system at your convenience. If you have any further questions or need assistance, please reach out to our support team.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/logout.md:25]:
```suggestion
The Logout option ensures a secure end to the user's session, redirecting the user to a dedicated logout page. From there, the user can conveniently navigate to different sections, including the home page, login page, and sign up page, using the provided buttons. Feel free to explore the various features available after logging out and return to the system at your convenience. If you have any further questions or need assistance, please reach out to our support team.
```

File: docs/src/user/manual/signup.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# Signup Page User Manual
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:1]:
```suggestion
# Sign Up Page User Manual
```

File: docs/src/user/manual/signup.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+## Overview
+
+The `Signup` page allows users to create a new account, providing access to personalised features and content on the platform. This guide will walk users through the steps to sign up, including filling out the required fields and creating their account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:5]:
```suggestion
The `Sign Up` page allows users to create a new account, providing access to personalised features and content on the platform. This guide will walk users through the steps to sign up, including filling out the required fields and creating their account.
```

File: docs/src/user/manual/signup.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+![signup](./img/signup-1.png)
+
+1. **Access Signup Page:** Click on the signup button available in the middle of the home page. Alternatively, locate and click on the login button in the top right corner of the page, where the user can find an option to create an account.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:9]:
```suggestion
1. **Access Sign Up Page:** Click on the `SIGN UP` button available in the middle of the home page. 
2. **Other Access to Sign Up Page:** Alternatively, locate and click on the `SIGN UP` button in the top right corner of the page, where the user can find an option to create an account.
```

File: docs/src/user/manual/signup.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    - Profile Picture: Optionally, upload a profile picture by selecting a file.
+
+    *Note:* Fields marked with * are required.
+
+2. **Create Account:** After filling out the required fields, click on the `Create Account` button.
+The system will process the user's information and create the user's account.
+
+## Conclusion
+
+Creating a new account is a simple and essential step to unlock personalised features on our platform. Follow these steps to complete the signup process, and the user be ready to explore and engage with our community. If the user encounters any issues or has questions, feel free to contact the support team for assistance.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:33]:
```suggestion
Creating a new account is a simple and essential step to unlock personalised features on our platform. Follow these steps to complete the sign up process, and the user be ready to explore and engage with our community. If the user encounters any issues or has questions, feel free to contact the support team for assistance.
```

File: docs/src/user/manual/signup.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+![signup](./img/signup-1.png)
+
+1. **Access Signup Page:** Click on the signup button available in the middle of the home page. Alternatively, locate and click on the login button in the top right corner of the page, where the user can find an option to create an account.
+
+## Signup Page
+
+![signup page](./img/signup-2.png)
+
+1. **Fill Out Signup Form:**
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:15]:
```suggestion
1. **Fill Out Sign up Form:**
```

File: docs/src/user/manual/signup.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+    The signup form includes the following fields:
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:17]:
```suggestion
    The sign up form includes the following fields:
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

File: docs/src/user/guide/forgot_password.md
Comment (ketankartoza) [reply docs/src/user/guide/forgot_password.md:1]: okay

File: docs/src/user/guide/logout.md
Comment (ketankartoza) [reply docs/src/user/guide/logout.md:1]: okay

File: docs/src/user/manual/signup.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+## Overview
+
+The `Sign Up` page allows users to create a new account, providing access to personalised features and content on the platform. This guide will walk users through the steps to sign up, including filling out the required fields and creating their account.
+
+![signup](./img/signup-1.png)
+
+1. **Access Sign Up Page:** Click on the `SIGN UP` button available in the middle of the home page. 
+2. **Other Access to Sign Up Page:** Alternatively, locate and click on the `SIGN UP` button in the top right corner of the page, where the user can find an option to create an account.
+
+## Signup Page
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:12]:
```suggestion
## Sign Up Page
```

File: docs/src/user/manual/signup.md [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+
+![signup page](./img/signup-2.png)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:14]:
```suggestion
![sign up page](./img/signup-2.png)
```

File: docs/src/user/manual/signup.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
+# Sign Up Page User Manual
+
+## Overview
+
+The `Sign Up` page allows users to create a new account, providing access to personalised features and content on the platform. This guide will walk users through the steps to sign up, including filling out the required fields and creating their account.
+
+![signup](./img/signup-1.png)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/signup.md:7]:
```suggestion
![sign up](./img/signup-1.png)
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
Minor changes

@Jeremy-Prior Done!!

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Pull request merged**